### DISCUSSION TITLE
Floccus on Apple Store: Can i get the app without an apply account?

### DISCUSSION THREAD

**Discussion by OnkelBruno** [floccusaddon/floccus#1522] [General]
Hi alltogether,

As me beeing very keen on saving my privacy, I found, that I have a big problem with downloading the app from the Apple Store. (Only Fake account with faked tel nummer)

Please dont get me wrong: Paying 2 € isnt a problem, but I have to enter some data, which compromizes my Privacy.

So, to bring it back to point: Is there a possibilty to get the app for ipados without giving my data to Apple (Paying is no problem)

Greetings from an paranoid admin from germany😊

**Comment by marcelklehr** [comment]
Mh, for now I think this might be hard. I might be able to put you down as a beta tester to prevent you from having to pay for it, if you give me the email addy for your apple id or whatever that is called.

**Reply by OnkelBruno** [reply]
Hey @marcelklehr , I have dropped you a Mail to your gmx-adress. Hoppefully it didnt go into your spam-folder 😜

**Reply by marcelklehr** [reply]
Yes, thank you, I've added you to the list of external testers, but wasn't able to send a build to testing, yet, for some reason

**Reply by marcelklehr** [reply]
You can try setting up the test pilot app on your device

**Reply by OnkelBruno** [reply]
Thank you! I think you meant "Testflight"? For using it, there is a invitation key needed. Can you please pass me one through mail?

**Reply by marcelklehr** [reply]
I've signed you up for TestFlight with floccus via your email already. I can't see a way to generate an invitation key anywhere

**Comment by OnkelBruno** [comment]
Thank you very much, Marcel. Unfortunately, the mail in my mailbox arrived belated. Now, there was a mail in my mailbox and I could install floccus. Thank you again. You've received a donation from my side :)

**Comment by NitroNils** [comment]
Won't this be solved with iOS 17.4? Oh, that only applies to Europe. Apple allowing other app stores.
